pkg/apis/servicecatalog/types.go

```diff
@@ -278,7 +279,9 @@ type BindingSpec struct {
 	// should be injected with the results of the binding.  Immutable.
 	AppLabelSelector metav1.LabelSelector
 
-	Parameters map[string]runtime.Object
+	// Parameters is a YAML representation of the properties to be
+	// passed to the underlying broker.
+	Parameters string `json:"parameters"`
```

Comment:
provision parameters are a JSON object, so this should be `map[interface{}]interface{}` to add specificity